### PR TITLE
Update arguments of `exec` command

### PR DESCRIPTION
**Pull request by MPvHarmelen**

### CODE REVIEW

**Comment by FichteFoll** [comment]
Thanks.

However, it looks like you introduced mixed indentation here. Our policy is to use spaces, but we haven't revised that in all files yet, so please stick to the file's standard for now.

**Comment by MPvHarmelen** [comment]
Ah, yes! I didn't notice, sorry. Is it now the way you would want it?

File: source/reference/commands.rst [hunk 1]
```diff
@@ -497,18 +497,24 @@ Commands
...
+	  ``cmd`` and ignores ``shell``.
 	- **file_regex** [String]
 	- **line_regex** [String]
 	- **working_dir** [String]
 	- **encoding** [String]
 	- **env** [{String: String}]
-	- **path** [String]
-	- **shell** [Bool]
+	- **quiet** [Bool]: If ``True`` information less running about prints the
+	  command.
```
Comment (FichteFoll) [inline source/reference/commands.rst]: This sentence makes no sense.

**Review by FichteFoll — CHANGES_REQUESTED** [review]

File: source/reference/commands.rst (hunk 1 above)
Comment (MPvHarmelen) [reply source/reference/commands.rst]: Yeah, I'm with you on that. However, I didn't write it (and did not want to spend the time trying to figure out what it should say). I could delete it if that makes everyone happier? :smile:

Comment (FichteFoll) [reply source/reference/commands.rst]: Change it to this: 

If True, don't print runtime information when the command fails or has a non-zero exit code.

**Comment by MPvHarmelen** [comment]
Thanks for the swift reply! I replaced the text and I (read: sublime-text) apparently also removed some trailing tabs.

**Pull request merged**